ibm/service/power/ibm_pi_constants.go

```diff
@@ -284,21 +294,25 @@ const (
 	Attr_SystemPoolName                              = "system_pool_name"
 	Attr_SystemPools                                 = "system_pools"
 	Attr_Systems                                     = "systems"
-	Attr_SysType                                     = "systype"
+	Attr_SysType                                     = "sys_type"
```

Comment:
I think the value of this constant can't be changed. It is being used as "systype" in the instance data source.
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5054/files